definitions/variable/emissions/carbon-management.yaml

```diff
@@ -8,25 +6,57 @@
       direct air carbon capture and storage (DACCS), and other CO2 removal exclusive of
       land-use change in these sectors
     unit: Mt CO2/yr
+    tier: 1
+    lower_bound: 0
+    components:
+      - Carbon Capture|Geological Storage
+      - Carbon Capture|Other Storage
+      - Carbon Capture|Leakage
+      - Carbon Capture|Utilization
+
+- Carbon Capture|{Carbon Capture Source}:
+    description: Captured carbon dioxide (CO2) from {Carbon Capture Source}
+    unit: Mt CO2/yr
+    tier: 1
+    lower_bound: 0
+    notes: Values should be reported as positive numbers.
+- Carbon Capture|Industrial Processes:
+    description: Captured carbon dioxide (CO2) from industrial processes
+      (e.g., cement production), not including energy use
+    unit: Mt CO2/yr
+    tier: 1
+    lower_bound: 0
+    notes: Values should be reported as positive numbers.
+
 - Carbon Capture|Geological Storage:
     description: Captured carbon dioxide (CO2) that is injected into subsurface geologic
       storage reservoirs (e.g., in situ mineralization).
     unit: Mt CO2/yr
+    tier: 1
+    lower_bound: 0
     notes: Captured CO2 that is reacted at the earth's surface (e.g., ex situ mineralization)
       should be reported under Carbon Capture|Other Storage
 - Carbon Capture|Other Storage:
     description: Other types of storage for captured carbon dioxide (CO2)
     unit: Mt CO2/yr
+    tier: 1
+    lower_bound: 0
 - Carbon Capture|Leakage:
     description: Captured carbon dioxide (CO2) that is vented, leaked, or re-emitted
```

Comment:
"or after injection into storage" should be removed here. Otherwise the sums don't add up. Need to think about where to add this, potentially as a separate variable? But this is not urgent.

Reply:
Done.